src/main/java/com/CourseScheduleBuilder/Repositories/UserRepo.java

```diff
@@ -10,9 +10,9 @@
 public interface UserRepo extends JpaRepository<User, String> {
 
 
-    User findByName(String name);
-
-    List<User> findByEmailAndYear(String email, int year);
+    User findByFirstName(String firstName);
+    User findByEmail(String email);
+    List<User> findByEmailAndPassword(String email, String password);
```

Comment:
Cool, h2 has support for serving based on multiple criteria?

Reply:
Yep you can be served based on any number of criteria